### PR TITLE
Merged Alignments: use config and actually select resources to be used

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

**Comment by ebelter** [comment]
@tmooney Plz review

**Comment by tmooney** [comment]
+1, seems good!

**Comment by ebelter** [comment]
@dufeiyu Plz review at your convenience.

File: etc/genome/spec/lsf_resource_merged_alignments.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This spec controls the LSF resource for instrument data merged alignments
+---
+default_value: "-R 'span[hosts=1] select[gtmp>90 && mem>16000] rusage[gtmp=90,mem=16000]' -M 16000000";
```
Comment (tmooney) [inline etc/genome/spec/lsf_resource_merged_alignments.yaml]: This semi-colon should be removed.

**Comment by ebelter** [comment]
CI, test this.

**Comment by dufeiyu** [comment]
+1

**Comment by jeldred** [comment]
merging after discussion in apipe weekly meeting

**Pull request merged**